force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.html

```diff
@@ -0,0 +1,87 @@
+<template>
+    <lightning-card  title="PD1 Calculator">
+      <!-- Process -->
+      <lightning-button 
+        variant="brand" 
+        label="Calculate" 
+        title="Calculate" 
+        slot="actions" 
+        onclick={calculateScore}
+      ></lightning-button>
+      <!-- Input Information  -->
+      <div class="slds-p-horizontal_small">
+        <lightning-input 
+          type="number" 
+          name="devFundamentals" 
+          label="Developer Fundamentals: 23%"
+          value={devFundamentalScore}
+          onchange={handleChange} 
+        ></lightning-input>
+        <lightning-input 
+          type="number" 
+          name="processAuto" 
+          label="Process Automation: 30%" 
+          value={processAutomationScore}
+          onchange={handleChange} 
+        ></lightning-input>
+        <lightning-input 
+          type="number" 
+          name="userInterface" 
+          label="User Interface: 25%" 
+          value={userInterfaceScore}
+          onchange={handleChange} 
+        ></lightning-input>
+        <lightning-input 
+          type="number" 
+          name="testDebugDeploy" 
+          label="Testing, Debugging, and Deployment: 22%" 
+          value={testingScore}
+          onchange={handleChange} 
+        ></lightning-input>
+      </div>
+      <!-- Display Results  -->
+      <div slot="footer">
+        <h1 class="slds-text-heading_large slds-text-title_caps resultText">Certification Results</h1>
+        <p class="slds-text-heading_medium">Certification Score: <span class="slds-text-color_destructive">{certificationScore}%</span></p>
+        <div>
+          <h1 class="slds-text-heading_large">Previous Attempt</h1>
+          <lightning-tabset>
+            <div for:each={attemptHistory} for:item="attempt" key={attempt.Id}>
+                <lightning-tab label={attempt.Id}>
+                  <c-score-information
+                    score={attempt.Score}
+                    attempt-id={attempt.Id}
+                    number-of-questions={numberOfQuestions}
+                    ondeleteattempt={deleteAttemptHandler}
+                  ></c-score-information>
+                </lightning-tab>
+            </div>
+          </lightning-tabset>
+        </div>
+        <template lwc:if={showResources}>
+          <ul class="slds-text-align_left slds-text-heading_small slds-var-p-top_medium slds-list_dotted slds-box">
+            <li class="resourceLi">
+              Resources:
+            </li>
+            <li>
+              <a href="https://trailhead.salesforce.com/en/credentials/platformdeveloperi">PD1 Exam Guide</a>
+            </li>
+            <li>
+              <a href="https://focusonforce.com/courses/salesforce-certified-platform-developer-1-exams/">Focus on Force</a>
+            </li>
+            <li>
+              <a href="https://courses.salesforcementor.com/p/salesforce-development-apex-fundamentals">Salesforce Mentor</a>
+            </li>
```

Comment:
Try moving these values into a js object and iterating over the values here to create the list items